src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int ARRAY_SIZE = 6;
+
+    public static void main(String[] args) {
+        int i;
+        ColorSupplier colorSupplier = new ColorSupplier();
+        Figure[] figureArray = new Figure[ARRAY_SIZE];
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        for (i = 0; i < figureArray.length / 2; i++) {
```

Comment:
You should create several random Figures, so you will use a loop. Please don't create a `new FigureSupplier()` inside the loop. However, in this case, the `figureSupplier` is created outside of the loop which is correct. Make sure 'i' is declared within the loop to avoid unnecessary scope extension.